README.md

```diff
@@ -32,9 +32,7 @@ Check out the [examples](examples) directory.
         jupyter-kernelspec install sparkmagic/kernels/sparkkernel
         jupyter-kernelspec install sparkmagic/kernels/pysparkkernel
         
-3. (Optional) Copy the example configuration file to your home directory. Look at the example config at https://github.com/jupyter-incubator/hdijupyterutils
-
-        touch ~/.sparkmagic/config.json
+3. (Optional) Modify the configuration file at ~/.sparkmagic/config.json. Look at the example config at https://github.com/jupyter-incubator/hdijupyterutils
```

Comment:
No, gets created during library run. I think users can do that mental leap of creating it? Am I the only one who always does that?